doc/configuration.rst

```diff
@@ -723,6 +731,11 @@ Environment variables
 
   * ``YORC_CONSUL_SSL_VERIFY``: Equivalent to :ref:`--consul_ssl_verify <option_consul_ssl_verify_cmd>` command-line flag.
 
+.. _option_consul_tls_handshake_timeout_env:
+
+  * ``YORC_CONSUL_TLS_HANDSHAKE_TIMEOUT``: Equivalent to :ref:`--consul_tls_handshake_timeout <option_consul_tls_handshake_timeout_cmd>` command-line flag.
```

Comment:
no way/need to give this timeout to terraform?

Reply:
Didn't find such config param and didn't need it in my test as the test was basic, just doing one deployment at a time. in this case, it was needed only at the preliminary phase of the deployment storing the deployment details in Consul, when Terraform is not yet involved.

Reply:
ok